planning/motion_velocity_planner/autoware_motion_velocity_planner_node/src/planner_manager.cpp

```diff
@@ -71,13 +71,67 @@ void MotionVelocityPlannerManager::update_module_parameters(
   for (auto & plugin : loaded_plugins_) plugin->update_parameters(parameters);
 }
 
+std::shared_ptr<DiagnosticStatus> MotionVelocityPlannerManager::makeDiagnostic(
```

Comment:
please use `snake_case` for the functions of this package.
```suggestion
std::shared_ptr<DiagnosticStatus> MotionVelocityPlannerManager::make_diagnostic(
```